containerd-shim/main.go

```diff
@@ -71,11 +72,7 @@ func start() error {
 	if err != nil {
 		return err
 	}
-	defer func() {
-		if err := p.Close(); err != nil {
-			logrus.Warn(err)
-		}
-	}()
+	defer p.Close()
```

Comment:
I think we should add it back. On containerd side we could just write those warning to the daemon logs?